pkg/apis/ml/v1beta1/extension_spec.go

```diff
@@ -63,14 +68,22 @@ func (a *ArangoMLExtensionSpec) GetInit() *ArangoMLExtensionTemplate {
 	return a.Init
 }
 
-func (a *ArangoMLExtensionSpec) GetStorage() *sharedApi.Object {
+func (a *ArangoMLExtensionSpec) GetStorage2() *sharedApi.Object {
```

Comment:
`GetStorage2`? Typo?

Reply:
Oh, yes! I just changed it to ensure all references are fixed